packages/dashboard/src/types/APIFilter.ts

```diff
@@ -1 +1,6 @@
-export type APIFilter = Record<'apiEndpoint' | 'valueSelector' | 'textSelector', string>
+type subGroupValues = Partial<Record<'subgroupValueSelector' | 'subgroupTextSelector', string>>
+
+export type APIFilter = Record<'apiEndpoint' | 'valueSelector' | 'textSelector', string> &
+  subGroupValues & {
+    values: [string, Record<string, string | number>[]]

```

Comment:
Why are we storing values on this type? the values should be calculated at runtime only. This type is meant for the configuration that is saved, not runtime data